applications/matter_bridge/src/bridged_device_types/generic_switch.cpp

```diff
@@ -68,6 +68,8 @@ CHIP_ERROR GenericSwitchDevice::HandleRead(ClusterId clusterId, AttributeId attr
 	switch (clusterId) {
 	case Clusters::Switch::Id:
 		return HandleReadSwitch(attributeId, buffer, maxReadLength);
+	case Clusters::Descriptor::Id:
+		return HandleReadDescriptor(attributeId, buffer, maxReadLength);
```

Comment:
Does this work? Descriptor cluster proves `AttributeAccesInterface` that overrides read regular read handlers. I think the right solution is to update `descriptor.cpp` in Matter (in the upstream the fix seems to be already done).

Reply:
Oh, Indeed it works this way. In Matter Bridge, there is no declaration of Descriptor's feature map attribute, so for now, it is enough to add it only, without changing the value because I can see that after enabling it the value read by chiptool is correct.